sky/authentication.py

```diff
@@ -488,4 +490,112 @@ def setup_kubernetes_authentication(config: Dict[str, Any]) -> Dict[str, Any]:
     file_mounts[PUBLIC_SSH_KEY_PATH] = PUBLIC_SSH_KEY_PATH
     config['file_mounts'] = file_mounts
 
+    sshjump_name = clouds.Kubernetes.SKY_SSH_JUMP_NAME
+    sshjump_image =  clouds.Kubernetes.SSH_JUMP_IMAGE
+
+    def _to_sshjump_pod_spec(name, image, secret):
```

Comment:
Can we define this pod spec as a jinja template (e.g., `templates/k8s_ssh_jump.j2`) that is filled in with `name`, `image` and `secret` through a function call? Same for the service spec below. Might be able to put them in the same file too.

Reply:
Yes, I pushed the changes.